datafusion/tests/sql.rs

```diff
@@ -47,6 +47,41 @@ use datafusion::{
 };
 use datafusion::{execution::context::ExecutionContext, physical_plan::displayable};
 
+/// Copied from datafusion/test/mod.rs
+/// TODO: consolidate
```

Comment:
This is basically a copy/paste from test/mod.rs -- I have plans to fix this in https://github.com/apache/arrow-datafusion/issues/743